src/rprblender/properties/render.py

```diff
@@ -429,7 +429,7 @@ def is_tile_render_available(self):
         ]
     if pyrpr2.enabled:
         render_quality_items += [
-            ('FULL2', "RPR 2.0 (Beta)", "Full render quality with RPR 2 (Beta)")
+            ('FULL2', "Full render quality using RPR 2, including hardware ray tracing support.")
```

Comment:
Name of option is missed here, which raises: ValueError: bpy_struct "RPR_RenderProperties" registration error: render_quality could not register